wordpress-mobile/WordPress-Android
# GB - Set editor preference to the remote site

Fixes #10192 by sending the editor pref to the remote site.  

- Editor preference is sent to the remote server each time the user switch the pref in App Settings
- The preference is sent to the remote server each time the editor is opened in the app. This was necessary since some users may have already switched to GB, and the new prompt dialog is not shown in these cases. We may want to revise this decision, and send data to the remote server when the dialog is shown only by putting a little bit of logic around it.
- The local app preference is set for all of the wpcom powered sites available in the app

FluxC PR: https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1305

To test:
- Switch the editor setting in the app
- Start the editor 
- On the wpcom backend for admins, check the section about the editor preferences in the RC card 

Update release notes:

- [ x ] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.
